### ISSUE TITLE
[Bug] ValueError: DataLayerWallet not available - spam in Log file

### ISSUE THREAD

**Issue by heget** [Chia-Network/chia-blockchain#18518]
### What happened?

After installed this latest version I'm getting a lot of spams in log file, see below.

### Version

2.4.3

### What platform are you using?

Windows

### What ui mode are you using?

GUI

### Relevant log output

```shell
2024-08-23T15:23:01.202 wallet chia.rpc.util              : WARNING  Error while handling message: Traceback (most recent call last):
  File "chia\rpc\util.py", line 84, in inner
  File "chia\rpc\wallet_rpc_api.py", line 4265, in dl_owned_singletons
  File "chia\wallet\wallet_state_manager.py", line 2624, in get_dl_wallet
ValueError: DataLayerWallet not available

2024-08-23T15:24:01.195 wallet chia.rpc.util              : WARNING  Error while handling message: Traceback (most recent call last):
  File "chia\rpc\util.py", line 84, in inner
  File "chia\rpc\wallet_rpc_api.py", line 4265, in dl_owned_singletons
  File "chia\wallet\wallet_state_manager.py", line 2624, in get_dl_wallet
ValueError: DataLayerWallet not available

2024-08-23T15:25:01.194 wallet chia.rpc.util              : WARNING  Error while handling message: Traceback (most recent call last):
  File "chia\rpc\util.py", line 84, in inner
  File "chia\rpc\wallet_rpc_api.py", line 4265, in dl_owned_singletons
  File "chia\wallet\wallet_state_manager.py", line 2624, in get_dl_wallet
ValueError: DataLayerWallet not available

2024-08-23T15:26:01.207 wallet chia.rpc.util              : WARNING  Error while handling message: Traceback (most recent call last):
  File "chia\rpc\util.py", line 84, in inner
  File "chia\rpc\wallet_rpc_api.py", line 4265, in dl_owned_singletons
  File "chia\wallet\wallet_state_manager.py", line 2624, in get_dl_wallet
ValueError: DataLayerWallet not available

2024-08-23T15:27:01.212 wallet chia.rpc.util              : WARNING  Error while handling message: Traceback (most recent call last):
  File "chia\rpc\util.py", line 84, in inner
  File "chia\rpc\wallet_rpc_api.py", line 4265, in dl_owned_singletons
  File "chia\wallet\wallet_state_manager.py", line 2624, in get_dl_wallet
ValueError: DataLayerWallet not available

2024-08-23T15:28:01.207 wallet chia.rpc.util              : WARNING  Error while handling message: Traceback (most recent call last):
  File "chia\rpc\util.py", line 84, in inner
  File "chia\rpc\wallet_rpc_api.py", line 4265, in dl_owned_singletons
  File "chia\wallet\wallet_state_manager.py", line 2624, in get_dl_wallet
ValueError: DataLayerWallet not available
```

**Comment by BrandtH22** [comment]
Hey @heget , as a heads up we can generally provide more timely, thorough support in our discord server (https://discord.gg/chia).

Can you verify that the datalayer features are disabled (if that is the intention) by going to the settings / DataLayer and ensuring the Enable DataLayer option is disabled? 

If you are disabling it now please close chia and restart it to ensure the changes take effect.

**Comment by heget** [comment]
I can confirm, after disabling the DataLayer then it's OK. 
But my farm was always set with dataLayer on. ;)

**Comment by altendky** [comment]
Looks like it was introduced in 2.4.3 rc2 via https://github.com/Chia-Network/chia-blockchain/pull/18166/files#diff-534c78c9c89cdf4166a424c59524af7b30e3b9a85cabd51b62cde9d8337cdf92R877.  Sorry for the hassle and thanks for pointing it out.  My first thought is that this should not be a warning level message, maybe info instead, and that it should be a regular log line not a traceback.

**Comment by wjblanke** [comment]
Fixed in 2.4.4
